lib/ast.js

```diff
@@ -345,7 +345,7 @@ var AST_Toplevel = DEFNODE("Toplevel", "globals", {
 var AST_Expansion = DEFNODE("Expansion", "expression", {
     $documentation: "An expandible argument, such as ...rest, a splat, such as [1,2,...all], or an expansion in a variable declaration, such as var [first, ...rest] = list",
     $propdoc: {
-        expression: "AST_Symbol the thing to be expanded"
+        expression: "[AST_Symbol] the thing to be expanded"
```

Comment:
> AFAICT array is marked as [AST_Node*]

But it's not a JS array of AST_Node either, it's either an AST_Array or a generic AST_Node.
```
expression: "AST_Node the thing to be expanded"
```